pkgs/tools/misc/youtube-dl/default.nix

```diff
@@ -10,33 +9,36 @@ stdenv.mkDerivation rec {
     sha256 = "1crfada7vq3d24062wr06sfam66cf14j06wnhg7w5ljzrbynvpll";
   };
 
-  buildInputs = [ python makeWrapper zip pandoc ];
+  nativeBuildInputs = [ pandoc ];
+
+  buildInputs = [ python makeWrapper zip ];
 
   patchPhase = ''
-    rm youtube-dl
+    rm ./youtube-dl
   '';
 
-  configurePhase = ''
-    makeFlagsArray=( PREFIX=$out SYSCONFDIR=$out/etc PYTHON=${python}/bin/python )
-  '';
+  makeFlags = [
+    "PREFIX=$(out)"
+    "SYSCONFDIR=$(out)/etc"
+    "PYTHON=${python.interpreter}"
+  ];
 
   postInstall = ''
-    # ffmpeg is used for post-processing and fixups
     wrapProgram $out/bin/youtube-dl --prefix PATH : "${ffmpeg}/bin"
   '';
 
-  meta = {
-    homepage = "http://rg3.github.com/youtube-dl/";
-    repositories.git = https://github.com/rg3/youtube-dl.git;
```

Comment:
The `repositories.git` attribute is used by the nixos-monitor, I believe, to check for updates to this package. I'm not sure removing it is a good idea.